spring-cloud-aws-sqs/src/main/java/io/awspring/cloud/sqs/annotation/SqsListener.java

```diff
@@ -131,6 +131,13 @@
 	 */
 	String maxMessagesPerPoll() default "";
 
+	/**
+	 * Specifies whether messages coming from FIFO queues should be grouped when retrieved by the container in listener
+	 * mode {@link io.awspring.cloud.sqs.listener.ListenerMode#BATCH}.
+	 * @return message grouping boolean
+	 */
+	boolean messageGrouping() default true;
```

Comment:
While I understand the practicality of having this configuration in the annotation, this seems to be a configuration for a reasonably specific case. We need to keep the annotation lean, otherwise we may end up with every container option in it, and it might become too polluted.

So my suggestion is we leave it only in the `ContainerOptions`.

What do you think?

Reply:
I agree, I asked myself the same question and then I said: "who can do more can do less" so I implemented it. I think it's great to keep the property only in `ContainerOptions` !